styles/cart.css

```diff
@@ -0,0 +1,36 @@
+.cart-wrapper {
+  margin: calc(var(--spacing-xl) * 2) 0;
+}
+
+.cart-head {
+  margin-bottom: calc(var(--spacing-xl) * 2);
+}
+
+.quantity-view {
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+
+  padding: 10px;
+
+  width: calc(var(--spacing-md) * 2);
+  height: calc(var(--spacing-md) * 2);
```

Comment:
appreciation:-  good to see this you gave a space which is related to border thn  length.

Reply:
Thanks @ashishkatiyar2404